src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -116,6 +116,9 @@ tokens {
     BINARY_DIGIT; ID_START; ID_PART; INT_LITERAL; LONG_LITERAL;
     FLOAT_LITERAL; DOUBLE_LITERAL; HEX_FLOAT_LITERAL; HEX_DOUBLE_LITERAL;
     SIGNED_INTEGER; BINARY_EXPONENT;
+
+    //Java 14 tokens
```

Comment:
I think @rnveach has a good comment here https://github.com/checkstyle/checkstyle/pull/8293#discussion_r448449960, though I see above that there is also "Tokens for Java 1.7" and "Tokens for Java 1.8" already... following that pattern, might use "Tokens for Java 14"

Reply:
I've just completely removed it as in https://github.com/checkstyle/checkstyle/pull/8293#discussion_r448449960. Fixed.